app/scripts/modules/appengine/serverGroup/configure/wizard/basicSettings.controller.ts

```diff
@@ -20,6 +23,8 @@ class AppengineServerGroupBasicSettingsCtrl implements IController {
               namingService: NamingService) {
     'ngInject';
 
+    $scope.containerImageUrlEnabled = SETTINGS.providers.appengine.defaults.containerImageUrlDeployments;
```

Comment:
In general, prefer putting values on `this` rather than `$scope` (the `BasicSettingsMixin` assumes that `$scope.command` is defined, which is why we do it here). 

You can initialize the value on the class as 

```
class AppengineServerGroupBasicSettingsCtrl {
  public containerImageUrlEnabled = SETTINGS.path.to.value;
  ...
}

```

On this template, `this` is aliased as `basicSettingsCtrl`.